src/main/scala/codacy/codenarc/CodeNarcOutput.scala

```diff
@@ -2,6 +2,7 @@ package codacy.codenarc
 
 import org.codenarc.results.{DirectoryResults, FileResults, Results}
 import org.codenarc.rule.Violation
+import scala.Option
```

Comment:
This is already [imported automatically](https://www.scala-lang.org/api/2.13.6/scala/Predef$.html) by `Predef.scala`